fkbeta/fkws/views.py

```diff
@@ -225,6 +225,39 @@ class VideoList(generics.ListCreateAPIView):
 
     `ordering` - Order results by specified field.  Prepend a minus for
                  descending order.  I.e. `?ordering=-id`.
+
+    `editor__username` - the username of the editor
+
+    `framerate` - the framerate in hz * 1000
+
+    `has_tono_records` - if the tono flag is set (true/false)
+
+    `is_filler` - if this is a filler video (true/false)
+
+    `name` - the exact name/title of the video
+
+    `name__icontains` - substring is part of name/title of the video
+
+    `organization__name` - the name of the organization this video is associated with
+
+    `played_count_web` - the number of times this video was played on the web
+
+    `played_count_web__gt` - greater than
+
+    `played_count_web__gte` - greater than or equal
+
+    `played_count_web__lt`  - less than
+
+    `played_count_web__lte` - less than or equal
+
+    `publish_on_web` - if this video is published ont the web (true/false)
+
+    `ref_url` - the exact reference url
+
+    `ref_url__startswith` - the reference url start with this string
+
+    `ref_url__icontains` - the reference url contain this string
+
```

Comment:
So django rest framework actually makes automatic UI for these in the browsable API. It would be possible to introspect these fields and just add all these fields automatically. I don't suggest you do it, but if we start adding more filter fields to more models, we probably should.

But I think just listing the fields should be enough to make it discoverable. It's a bit much right now. People have to understand what the fields are anyway (and the `__startswith` etc is self-documenting).

Reply:
Perhaps a bit much, but I guess it depend on how deep in the code the reader is.  We can always reduce the amount later or generate it automatically. :)